test/unit/ItemFilters.test.js

```diff
@@ -181,7 +181,7 @@ describe('ItemFilters', () => {
       expect(itemFilterInfo.find('span').text()).to.equal("Filtered by format: 'Text', 'PRINT'");
     });
     it('should display "1 Result Found"', () => {
```

Comment:
same here. also maybe including the case being tested. ie 'there is one electronic item and no physical items'